Scripts/Runtime/Entities/TaskDriver/System/ContextTaskDriverSystemWrapper.cs

```diff
@@ -34,16 +34,16 @@ public ContextTaskDriverSystemWrapper(AbstractTaskDriverSystem taskDriverSystem,
             m_ContextTaskDriver = contextTaskDriver;
         }
 
-        public ISystemDataStream<TInstance> CreateDataStream<TInstance>(CancelRequestBehaviour cancelRequestBehaviour = CancelRequestBehaviour.Delete) 
+        public ISystemDataStream<TInstance> CreateDataStream<TInstance>(CancelRequestBehaviour cancelRequestBehaviour = CancelRequestBehaviour.Delete, string uniqueContextIdentifier = null) 
             where TInstance : unmanaged, IEntityProxyInstance
         {
-            return m_TaskDriverSystem.CreateDataStream<TInstance>(m_ContextTaskDriver, cancelRequestBehaviour);
+            return m_TaskDriverSystem.CreateDataStream<TInstance>(m_ContextTaskDriver, cancelRequestBehaviour, uniqueContextIdentifier);
         }
 
-        public ISystemEntityPersistentData<T> CreateEntityPersistentData<T>() 
+        public ISystemEntityPersistentData<T> CreateEntityPersistentData<T>(string uniqueContextIdentifier) 
```

Comment:
Why isn't there a default value for the UID here?

Reply:
There should be and now there is. Done.